### DISCUSSION TITLE
ttkbootstrap seems to be not compatible with tkCalendar

### DISCUSSION THREAD

**Discussion by alvgin** [israel-dryer/ttkbootstrap#572] [Q&A]
I'm a bit new on ttkbootstrap. I really think it look nicer than normal tkinter. 
So I try to migrate my programs from tkinter to ttkbootstrap widgets. ... but I found and issue when migrate a Calendar that works with   tkCalendar library.
When I import ttkbootstrap for some reason Calendar does not work. It seems to be not compatible. 
For the other hand I could use the DatePicker of ttkbootstrap but the functionality I need, which is mark some branch of dates in the calendar, seems to be not implemented yet on ttkbootstrap Calendar. 
Any idea on that? 
I'm using the newest libraries and even Python 3.10

**Comment by JannekKn** [comment]
did you find any solution to this? have the same issue right now

**Comment by TheDemonGuard** [comment]
@alvgin 
I have create a bug report regarding this. #806 
You can add any more information if necessary to this bug report.

> I'm a bit new on ttkbootstrap. I really think it look nicer than normal tkinter. So I try to migrate my programs from tkinter to ttkbootstrap widgets. ... but I found and issue when migrate a Calendar that works with tkCalendar library. When I import ttkbootstrap for some reason Calendar does not work. It seems to be not compatible. For the other hand I could use the DatePicker of ttkbootstrap but the functionality I need, which is mark some branch of dates in the calendar, seems to be not implemented yet on ttkbootstrap Calendar. Any idea on that? I'm using the newest libraries and even Python 3.10

**Reply by TheDemonGuard** [reply]
@alvgin 
This is fixed #1061 and available from ttkbootstrap 1.20.3.
Is it possible to try again.

> @alvgin I have create a bug report regarding this. #806 You can add any more information if necessary to this bug report.
> 
> > I'm a bit new on ttkbootstrap. I really think it look nicer than normal tkinter. So I try to migrate my programs from tkinter to ttkbootstrap widgets. ... but I found and issue when migrate a Calendar that works with tkCalendar library. When I import ttkbootstrap for some reason Calendar does not work. It seems to be not compatible. For the other hand I could use the DatePicker of ttkbootstrap but the functionality I need, which is mark some branch of dates in the calendar, seems to be not implemented yet on ttkbootstrap Calendar. Any idea on that? I'm using the newest libraries and even Python 3.10
